### ISSUE TITLE
Switch tagging to semantic versioning without project name

### ISSUE THREAD

**Issue by isuftin** [USGS-CIDA/Water-Auth-Server#39]
We should probably switch the tagging for this to be (for example) 0.0.2 instead of waterauthserver-0.0.2

This is easier to deal with on the Jenkins end when building Docker images and is standard for Java projects.

**Comment by zmoore-usgs** [comment]
Agreed; I don't remember why we went the the versioning scheme that we did. May have been related to how our CI/CD jobs were finding the artifacts.
